### PR TITLE
blog post about merger

### PR DESCRIPTION
**Pull request by raynamharris**
cc @tracykteal

### CODE REVIEW

File: _posts/2017/09/2017-09-02-merger.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+authors: ["Rayna Harris", "Tracy Teal"]
+title: "Joint future for Software Carpentry and Data Carpentry"
+date: 2017-09-02
+time: "00:00:00"
+category: [ "Steering Committee", "Community", "Data Carpentry"]
+---
+
+*“If you want to go fast, go alone. If you want to go far, go together.”*
+
+Software Carpentry and Data Carpentry are currently independent organizations with their own fiscal sponsorship, Steering Committees, governance model, and bank accounts. However, as is perhaps no surprise the organizations' operations have evolved to share memberships, infrastructure for workshop coordination, an instructor training program, and even some staff members. This 'separate but collaborative' organizational structure has allowed us to build a shared community of instructors with more than 1000 certified instructors and 47 current Member Organizations around the world.
```
Comment (karinlag) [inline _posts/2017/09/2017-09-02-merger.md]: Is it obvious for most people that these orgs are really sister orgs? I've been involved for a long time, and I know the "origin story", so to speak, but just wondering what you guys think. If you don't think the shared origins are obvious to all, I suggest putting in a sentence or two at the beginning that explains that.

**Comment by karinlag** [comment]
I think this looks good, but maybe add something about the fact that SWC and DC will remain as the lesson organizations and lesson brands? Right now it just says "associated lesson organizatons", which might not be instantanously recognizable/understandable for somebody who hasn't followed the process closely.

**Comment by raynamharris** [comment]
Thanks @karinlag. I attempted to make implicit more explicit.

**Comment by jduckles** [comment]
@raynamharris Please let me know when you'd like this to be merged.

**Comment by mkuzak** [comment]
thanks @raynamharris looks great

**Comment by gvwilson** [comment]
Hi @raynamharris - thanks very much for putting this together. May I request two changes to the timeline diagram?

1. Can you please remove the "Dr." in front of my name?

2. Can you please add "2005: Course materials open sourced with support from the Python Software Foundation." ? That was what started bringing SWC to a wider audience, and we wouldn't exist without it.

Thanks,
Greg

**Pull request merged**